packages/core-components/src/components/Select/static/ClosedDropdown.tsx

```diff
@@ -42,7 +43,7 @@ const ClosedDropdown = () => {
     >
       <path
         d="M7.5 8L6 9.5L12.0703 15.5703L18.1406 9.5L16.6406 8L12.0703 12.5703L7.5 8Z"
-        fill="#616161"
+        fill={grey[700]}
```

Comment:
Think this one makes sense for example

Reply:
Do you mean how it was or how I changed it?

Reply:
How you changed it